Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM 169942020521.dkr.ecr.eu-west-1.amazonaws.com/base/node:14-alpine-builder
```

Comment:
Do you know where this number comes from? That looks like a date at the end: 2020-05-21